dll/win32/msgina/shutdown.c

```diff
@@ -538,6 +561,89 @@ ShutdownDialog(
     return ret;
 }
 
+/*************************************************************************
+ * SetRegistryToBootInNTNativeMode
+ *
+ * NOTES
+ *     Used to append the NT Native shell executable to the BootExecute
+ *     registry value then triggers a restart in order to boot into it.
+ */
+VOID SetRegistryToBootInNTNativeMode(VOID)
+{
+    // prepare the registry for booting into the NT Native Shell
+    LONG lRet;
+    HKEY hKey;
+    DWORD dwType = 0, dwCurrentValueSize = 0;
+    WCHAR *pszCurrentValue = NULL, *pszFinalValue = NULL;
+    UINT cchLatest = 0;
+    const UINT wcharSize = sizeof(WCHAR);
```

Comment:
can just use `sizeof(WCHAR)` in your code, will be replaced with a const by the compiler.